packages/decision-tree/src/hooks/useHandleAnswers/useHandleAnswers.ts

```diff
@@ -30,3 +58,19 @@ export const useHandleAnswers = (callback: UseSetupCallbackCB): useHandleActionR
 
   return { state, answersDispatch: dispatch, submitAnswer, submitIDKAnswer };
 };
+
+const isAnswerFilled = (answer: AnswerModel | null): boolean => {
+  if (!answer) return false;
+
+  switch (answer.type) {
+    case 'singleChoice':
+    case 'numeric':
+      return !!answer.value;
+    case 'multipleChoice':
+      return answer.values.length > 0;
+    case 'height':
+      return !!answer.feet || !!answer.inches;
+    default:
```

Comment:
is this needed?

Reply:
yes, I don't remember the case, but I did that on purpose

Reply:
please double check why this is needed, it is better if we can avoid having a `default` because that will _warn_ us about missing cases handling when we add new question types

Reply:
I think it was something about going back and forth in the questions, but it looks good right now. I can't find the case, everything seems to work without it, so let's remove it and if we run into a reason we will assess then.